---
title: "Guilherme Marcelino Viana de Siqueira"
output:
  html_document:
    css: "cv-style.css"
---

```{r load_packages, message=FALSE, warning=FALSE, include=FALSE}
library(fontawesome)
```

<div class="neat-icons">
<p> `r fa(name = "location-dot")` Berkeley, CA, United States | Ribeirão Preto, SP, Brazil</p>

<p> `r fa(name = "phone")` +1 510 423-2246</p>

<p> `r fa(name = "envelope")` [viana.guilherme\@proton.me](mailto:guiviana@proton.me){.email}</p>

<p> `r fa(name = "graduation-cap")` [Google Scholar](https://scholar.google.com.br/citations?user=m-KrmPEAAAAJ&hl=en>Scholar)</p>

<p> `r fa(name = "github")` [Github](https://github.com/viana-guilherme)</p>

<p> `r fa(name = "linkedin")` [Linkedin](https://www.linkedin.com/in/guilheviana/)
</div>

<hr class="pretty-hr">

## Summary

I am an engaged and motivated doctoral candidate with more than 8 years of experience in research who thrives in interdisciplinary teams. I'm expected to defend my thesis at the latest in February 2025, and I'm looking forward to getting more experience as a scientist in the private sector after graduating.

I have extensive wet-lab experience in standard Molecular Biology and Microbiology techniques (media preparation, nucleic acid purification, polymerase chain reaction, restriction cloning, Gibson assembly, microplate assays, flow cytometry, and shake-flask culturing of bacteria) and sequencing (especially Nanopore sequencing).

I am also proficient in several frameworks and computational pipelines for the analysis of different types of biological data (including genomic, transcriptomic and proteomic datasets) using R (Tidyverse packages, DESeq2, vegan, dada2 and flowCore), Python (Biopython, numpy, pandas and matplotlib) and bash (general scripting and CLI Bioinformatics tools such as the SRA Toolkit, Trimmomatic, Kallisto, RSEM, Minimap2 and Guppy).

## Experience

As a graduate student, I have been working with leveraging the stress response of industrially-relevant organisms for the bioproduction of chemicals. This included devising genetic parts for improving host fitness under stress and expanding the limited toolbox for reliable genetic manipulation of non-conventional microbial hosts.

Listed below are the main objectives and achievements of the projects I have developed during this period:

### Research Associate

<subtle>Lawrence Berkeley National Laboratory \| Joint BioEnergy Institute (JBEI) (Feb 2023 - ongoing)</subtle>

#### Improving biofuel production from lignocellulosic biomass in *Pseudomonas putida*

At JBEI I have been working with the tolerization of *P. putida* against growth inhibitors present in lignocellulosic biomass for improved biofuel production. Utilizing a custom-tailored deep-well plate protocol, I have been able to isolate 5 tolerized strains with **enhanced growth and biofuel production metrics** in mock hydrolysate culture media when compared to their parental strain. Results deriving from this work are being compiled in a manuscript to be submitted for publication later this year.

### Graduate Student

<subtle>University of São Paulo (2018 - ongoing)</subtle>

#### Searching for new molecular tools to enhance *Pseudomonas putida* robustness to abiotic stresses

In my ongoing PhD project, we intend to develop and characterize new tools for engineering *Pseudomonas putida* as a host suitable for bioproduction. Our approach to this project is multi-layered, and comprises several smaller objectives:

-   I have designed the **pVANT** family of vectors, which are based on the Standard European Vector Architecture (SEVA). pVANT reporter vectors have been cloned in 6 different species of bacteria, including laboratory strains and environmental isolates, and exhibit **improved fluorescence when compared to their SEVA counterparts** while retaining all of the advantages of the European standard such as modularity, small size and broad host range applicability. Part of this work has been published in ACS Synthetic Biology in 2023 (10.1021/acssynbio.3c00098).

-   I have analyzed publicly-available raw RNAseq data from NCBI's GEO repository to compile a database of Pseudomonas transcriptomic data in over a dozen conditions. This database, has been used a starting point to search for new biological parts under stress conditions. We've so far identified eight putative **novel stress-responsive promoters** and three novel **tolerance-promoting proteins**, which are being characterized by our group.

#### Expanding acid resistance in bacteria using synthetic circuits

While obtaining my Masters degree, I led a project in which we designed synthetic operons, aiming to build an **optimized gene cluster that conferred *Escherichia coli* the ability to thrive in low pH**. I constructed a library of 27 acid resistance clusters (ARCs) and characterized the resulting strains regarding survival and growth under extreme acidic conditions. This work was published in ACS Synthetic Biology (10.1021/acssynbio.0c00089) and earned a [research highlight](https://www.nature.com/articles/s41589-020-0580-3) in Nature Chemical Biology in 2020.

### Other projects

#### Nanopore sequencing

Since 2020, I have been actively learning and contributing to projects using Nanopore sequencing platforms, in particular multiplexed microbial community sequencing projects using the SQK-16S024 kit on Flongle flowcells. I have experience in library preparation using rapid barcoding kits and ligation sequencing kits, as well in the monitoring and analysis of sequencing data using custom pipelines and tools such as MinKNOW, Guppy, NanoFilt and Minimap2. As a result of these projects I have published two papers.

#### mipreadr: An R package for the analysis of microbial growth data

I recently started working on the development of my first proper R package, mipreadr, a suite of functions for importing and processing microplate reader data into R for downstream analyses. While still under development, this tool is already being employed internally in our lab to generate robust and reproducible analyses of microbial growth and fluorescence assays, simplifying complex scripting. More information about mipreadr can be found in its [GitHub repository](https://github.com/viana-guilherme/mipreadr).

## Education

|                                                                                                                |      Year      |
|:-----------------------------------------------------|:----------------:|
| <subtle>University of São Paulo, Ribeirão Preto - São Paulo, Brazil</subtle> <br> **PhD in Molecular Biology** | 2020 - current |
| <subtle>University of São Paulo, Ribeirão Preto - São Paulo, Brazil</subtle> <br> **MSc. in Biochemistry**     |  2018 - 2020   |
| <subtle>University of Brasília, Brasília - Distrito Federal, Brazil</subtle> <br> **BSc. in Biotechnology**    |  2014 - 2017   |

## Publications

-   **de Siqueira, Guilherme Marcelino Viana** and María-Eugenia Guazzaroni. 'Host-Dependent Improvement of GFP Expression in Pseudomonas Putida KT2440 Using Terminators of Metagenomic Origin'. ACS Synthetic Biology 12, no. 5 (19 May 2023): 1562--66. <https://doi.org/10.1021/acssynbio.3c00098>.

-   Mancilio, Lucca Bonjy Kikuti, Guilherme Augusto Ribeiro, Erica Janaina Rodrigues de Almeida, **Guilherme Marcelino Viana de Siqueira**, Rafael Silva Rocha, María-Eugenia Guazzaroni, Adalgisa Rodrigues De Andrade, and Valeria Reginatto. 'Adding Value to Lignocellulosic Byproducts by Using Acetate and P-Coumaric Acid as Substrate in a Microbial Fuel Cell'. Industrial Crops and Products 171 (November 2021): 113844. <https://doi.org/10.1016/j.indcrop.2021.113844>.

-   **de Siqueira, Guilherme Marcelino Viana**, Felipe Marcelo Pereira-dos-Santos, Rafael Silva-Rocha, and Maria-Eugenia Guazzaroni. 'Nanopore Sequencing Provides Rapid and Reliable Insight into Microbial Profiles of Intensive Care Units'. Preprint. Microbiology, 14 May 2021. <https://doi.org/10.1101/2021.05.14.444165>.

-   Mancílio, Lucca Bonjy Kikuti, Guilherme Augusto Ribeiro, Erica Mendes Lopes, Luciano Takeshi Kishi, Leonardo Martins-Santana, **Guilherme Marcelino Viana de Siqueira**, Adalgisa Rodrigues Andrade, María-Eugenia Guazzaroni, and Valeria Reginatto. 'Unusual Microbial Community and Impact of Iron and Sulfate on Microbial Fuel Cell Ecology and Performance'. Current Research in Biotechnology 2 (November 2020): 64--73. <https://doi.org/10.1016/j.crbiot.2020.04.001>.

-   **de Siqueira, Guilherme Marcelino Viana**, Rafael Silva-Rocha, and María-Eugenia Guazzaroni. 'Turning the Screw: Engineering Extreme PH Resistance in Escherichia Coli through Combinatorial Synthetic Operons'. ACS Synthetic Biology 9, no. 6 (19 June 2020): 1254--62. <https://doi.org/10.1021/acssynbio.0c00089>.

-   Ribeiro, Lucas Ferreira, Vanesa Amarelle, Luana de Fátima Alves, **Guilherme Marcelino Viana de Siqueira**, Gabriel Lencioni Lovate, Tiago Cabral Borelli, and María-Eugenia Guazzaroni. 'Genetically Engineered Proteins to Improve Biomass Conversion: New Advances and Challenges for Tailoring Biocatalysts'. Molecules 24, no. 16 (January 2019): 2879. <https://doi.org/10.3390/molecules24162879>.

-   Alves, Luana de Fátima, Cauã Antunes Westmann, Gabriel Lencioni Lovate, **Guilherme Marcelino Viana de Siqueira**, Tiago Cabral Borelli, and María-Eugenia Guazzaroni. 'Metagenomic Approaches for Understanding New Concepts in Microbial Science'. International Journal of Genomics 2018 (23 August 2018): 1--15. <https://doi.org/10.1155/2018/2312987>.

## Grants and awards

**2023** - Research Internship Abroad (BEPE) fellowship awarded by the The São Paulo Research Foundation (FAPESP)

**2019** - "Best poster" category award in the III National Meeting of Biotechnological and Agroindustrial Chemistry. EnqBiotec. Brazil.

**2015** - "Young talents for Science" fellowship by the brazilian federal funding agency CAPES (Coordenação de Aperfeiçoamento de Pessoal de Nível Superior).
